integration/util.go

```diff
@@ -103,6 +103,8 @@ func (k *NSKubernetesClient) WaitForPodsReady(podNames ...string) {
 	waitLoop:
 		select {
 		case <-ctx.Done():
+			k.printDiskFreeSpace()
+			k.debug("nodes")
```

Comment:
Does `k.debug("nodes")` add anything? After all, the nodes are already unconditionally printed in `make integration` (https://github.com/GoogleContainerTools/skaffold/pull/2406/files#diff-b67911656ef5d18c4ae36cb6741b7965R112).

Reply:
When the `TestDebug` used to fail, I could see an issue with the k8s nodes.
I added those logs before the test suite and in case of an error, to detect cases where the kind cluster might get broken after running a few tests.

Reply:
Ok, that makes sense.